alws/releases.py

```diff
@@ -110,8 +79,11 @@ async def get_release_plan(db: Session, build_ids: typing.List[int],
                            base_dist_name: str, base_dist_version: str,
                            reference_dist_name: str,
                            reference_dist_version: str) -> dict:
-    # FIXME: put actual endpoint to use
-    endpoint = f'/api/v1/distros/{reference_dist_name}/' \
+    clean_ref_dist_name = re.search(
+        r'(?P<dist_name>[a-z]+)', reference_dist_name,
+        re.IGNORECASE).groupdict().get('dist_name')
+    clean_ref_dist_name_lower = clean_ref_dist_name.lower()
+    endpoint = f'/api/v1/distros/{clean_ref_dist_name}/' \
                f'{reference_dist_version}/projects/'
     packages = []
     repo_name_regex = re.compile(r'\w+-\d-(?P<name>\w+(-\w+)?)')
```

Comment:
Module `re` compiles and caches regexps inside itself, so you don't need do it. FYI